Tests/Unit/NetworkServer/ConcentratorDeduplicationTest.cs

```diff
@@ -20,24 +22,34 @@ public sealed class ConcentratorDeduplicationTest : IDisposable
 
         private readonly LoRaDevice loRaDevice;
         private readonly SimulatedDevice simulatedABPDevice;
+        private readonly LoRaPayloadData dataPayload;
         private readonly SimulatedDevice simulatedOTAADevice;
+        private readonly LoRaPayloadJoinRequest joinPayload;
         private readonly WaitableLoRaRequest dataRequest;
         private readonly WaitableLoRaRequest joinRequest;
 
+        public static TheoryData<byte[]?> TestRawMessages =>
+            new TheoryData<byte[]?>
+            {
+                null,
+                Array.Empty<byte>(),
+                new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 }
+            };
```

Comment:
nit: Can be made a static field and the constructed type can be omitted for brevity:

```suggestion
        public static readonly TheoryData<byte[]?> TestRawMessages = new()
        {
            null,
            Array.Empty<byte>(),
            new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 }
        };
```

Reply:
I would recommend moving this field closer to the `CreateKeyMethod_Should_Include_All_Properties_For_Different_RawMessages` test where it's used.

Reply:
Thanks, this exact code was removed but we have 2 other instances that do the same where I took your suggestion about making it a static field and omitting the constructed type. Didn't move it closer to the test though because I think it reads better to have all the tests together and last the supporting data if you need to look up/amend them.